python_modules/libraries/dagster-duckdb/dagster_duckdb/type_handlers.py

```diff
@@ -0,0 +1,135 @@
+from pathlib import Path
+from typing import Union, cast
+
+import duckdb
+
+from dagster import InputContext, OutputContext
+from dagster import _check as check
+
+from .io_manager import DbTypeHandler
+
+
+class DuckDBPandasTypeHandler(DbTypeHandler):
```

Comment:
The snowflake io manager breaks all of it's type handlers out into new libraries so that a user doesn't need to install a library they wont use. I might need to do that here as well

Reply:
If those libraries aren't already requirements of duckdb, then I think yes.  Here's the discussion we had about this for the snowflake IO manager: https://github.com/dagster-io/dagster/pull/7726#issuecomment-1119047634.

Reply:
ok - split everything out into new libraries